services/requester/requester.go

```diff
@@ -468,14 +468,6 @@ func cadenceStringToBytes(value cadence.Value) ([]byte, error) {
 // TODO(m-Peter): Consider moving this to flow-go repository
 func getErrorForCode(errorCode uint64) error {
 	switch evmTypes.ErrorCode(errorCode) {
-	case evmTypes.ValidationErrCodeInvalidBalance:
```

Comment:
why was this removed?

Reply:
These 4 errors got removed from upstream in this commit: https://github.com/onflow/flow-go/commit/53b6c8256b9082958162de419ddd309b6d8fffd7